bundles/framework/myplaces3/MyPlacesList.jsx

```diff
@@ -0,0 +1,116 @@
+            dataIndex: ['properties', 'name'],
+            sorter: (a, b) => a.properties.name.localeCompare(b.properties.name),
+            defaultSortOrder: 'ascend',
+            render: (title, item) => {
+                const shape = getGeometryIcon(item.geometry);
+                return (
+                    <a onClick={() => showPlace(item.geometry, item.categoryId)}>
+                        <div className={`icon myplaces-${shape}`} />
+                        <span>{title}</span>
+                    </a>
+                );
+            }
+        },
+        {
+            align: 'left',
+            title: <Message messageKey='tab.grid.desc' bundleKey={LOCALE_KEY} />,
+            dataIndex: ['properties', 'description'],
+            sorter: (a, b) => a.properties.description.localeCompare(b.properties.description)
+        },
+        {
+            align: 'left',
+            title: <Message messageKey='tab.grid.createDate' bundleKey={LOCALE_KEY} />,
+            dataIndex: 'createDate',
+            sorter: getSorterFor('createDate')
+        },
+        {
+            align: 'left',
+            title: <Message messageKey='tab.grid.updateDate' bundleKey={LOCALE_KEY} />,
+            dataIndex: 'updateDate',
+            sorter: getSorterFor('updateDate')
+        },
+        {
+            align: 'left',
+            title: <Message messageKey='tab.grid.measurement' bundleKey={LOCALE_KEY} />,
+            dataIndex: 'measurement',
+            sorter: getSorterFor('measurement')
+        },
+        {
+            align: 'left',
+            title: <Message messageKey='tab.grid.actions' bundleKey={LOCALE_KEY} />,
+            dataIndex: 'id',
+            render: (title, item) => {
+                return (
+                    <ToolsContainer>
+                        <div className='icon t_edit' onClick={() => handleEdit(item)}>
+                            <EditOutlined style={ EDIT_ICON_STYLE } />
+                        </div>
+                        <Confirm
+                            title={<Message messageKey='tab.notification.delete.confirm' messageArgs={{ name: item.properties.name }} bundleKey={LOCALE_KEY} />}
+                            onConfirm={() => handleDelete(item)}
+                            okText={<Message messageKey='buttons.ok' bundleKey={LOCALE_KEY} />}
+                            cancelText={<Message messageKey='buttons.cancel' bundleKey={LOCALE_KEY} />}
+                            placement='bottomLeft'
+                        >
+                            <div className='icon t_delete'><DeleteOutlined style={ DELETE_ICON_STYLE } /></div>
+                        </Confirm>
+                    </ToolsContainer>
+                );
+            }
+        }
+    ];
+
+    return (
+        <StyledTable
+            columns={columnSettings}
+            dataSource={data.map(item => ({
+                key: item.id,
+                ...item
+            }))}
```

Comment:
Spread feature properties to table items and save original item under _place if it's needed. Then you can use the getSorterFor() for everything:
```suggestion
            dataSource={data.map(item => ({
                key: item.id,
                ...item.properties,
                _place: item
            }))}
```